go.mod

```diff
@@ -6,5 +6,5 @@ require (
 	github.com/json-iterator/go v1.1.10
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/tools v0.1.0 // indirect
+// golang.org/x/tools v0.1.0 // indirect
```

Comment:
Why is this commented out?

Reply:
Cos my editor was complaining about it but unsure if it's better to just delete.

Reply:
Removed it